.github/workflows/comparison-results.yml

```diff
@@ -0,0 +1,145 @@
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Folder to store reports
+        run: mkdir ${GITHUB_WORKSPACE}/reports
+
+      - name: Download CPU and Memory usage capture tool
+        run: mkdir ${GITHUB_WORKSPACE}/tools && curl https://raw.githubusercontent.com/Privado-Inc/privado-core-results-monitoring/main/capture_usage.sh --output ${GITHUB_WORKSPACE}/tools/capture_usage.sh
+
+      - name: Start monitoring cpu and memory usage
+        run: bash ${GITHUB_WORKSPACE}/tools/capture_usage.sh &
+
+      # Setup python 3.8 in the pipeline
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
+
+      # Setup privado environment in the pipeline
+      - name: Setup Privado Env
+        run: curl -o- https://raw.githubusercontent.com/Privado-Inc/privado-cli/main/install.sh | bash
+
+      - name: Download the comparison tool
+        run: curl https://raw.githubusercontent.com/Privado-Inc/privado-core-results-monitoring/main/compare.py --output ${GITHUB_WORKSPACE}/tools/compare.py
+
+
+      # For every repository, clone, scan and store the generated JSON
+      # For Privado-Demo/privado-analytics
+      - name: Cloning Privado-Demo/privado-analytics
+        uses: actions/checkout@v2
+        with:
+          owner: "Privado-Demo"
+          repository: "Privado-Demo/privado-analytics"
+          path: ./repos/privado-analytics
+
+      # cannot simulate restarting the terminal behavior in the pipeline, so navigated to the installation path to run the tool
+      - name: Scan karan-batavia/privado-analytics (stable) using privado scan
+        run: ~/.privado/bin/privado scan --skip-upload ${GITHUB_WORKSPACE}/repos/privado-analytics
+
+      - name: Store privado-analytics.json (stable)
+        run: mkdir ${GITHUB_WORKSPACE}/stable_results && cat ${GITHUB_WORKSPACE}/repos/privado-analytics/.privado/privado.json > ${GITHUB_WORKSPACE}/stable_results/privado-analytics.json
+
+      - name: Scan karan-batavia/privado-analytics (dev) using privado scan
+        run: cd ~/.privado/bin && PRIVADO_DEV=1 ./privado scan --overwrite --skip-upload ${GITHUB_WORKSPACE}/repos/privado-analytics
+
+      - name: Store privado-analytics.json (dev)
+        run: mkdir ${GITHUB_WORKSPACE}/dev_results && cat ${GITHUB_WORKSPACE}/repos/privado-analytics/.privado/privado.json > ${GITHUB_WORKSPACE}/dev_results/privado-analytics.json
+
+      - name: Compare the two results
+        run: cd ${GITHUB_WORKSPACE}/tools && python ./compare.py ${GITHUB_WORKSPACE}/stable_results/privado-analytics.json ${GITHUB_WORKSPACE}/dev_results/privado-analytics.json
+
+      # For Privado-Demo/privado-accounts-api
+      - name: Cloning Privado-Demo/privado-accounts-api
+        uses: actions/checkout@v2
+        with:
+          owner: "Privado-Demo"
+          repository: "Privado-Demo/privado-accounts-api"
+          path: ./repos/privado-accounts-api
```

Comment:
I suggest we bring these repos here soon, prefixed with `demo-` pr `sample-`, like `demo-privado-accounts-api`.

Reply:
We can do this later too

Reply:
Will do, till then can't add env variable for the organization name as well